pkgs/development/libraries/qt-5/modules/qttools.nix

```diff
@@ -29,6 +35,12 @@ qtModule {
     })
   ];
 
+  postPatch = ''
+    # llvm-config --includedir gives libllvm includedir, looks for libclang header there
+    substituteInPlace src/qdoc/configure.pri \
+      --replace-fail "\''$\''$CLANG_INCLUDEPATH/clang-c" "${lib.getDev llvmPackages.libclang}/include/clang-c"
```

Comment:
Feels a bit weird to move sed calls to a patch and then add another sed call?

Reply:
Also moved to a patch now.

Reply:
Yeah, I definitely prefer that.